apps/web/src/pages/Communities/CommunityMembersPage/components/useAvailableRoles.ts

```diff
@@ -35,7 +33,7 @@ const useAvailableRoles = (): UseAvailableRolesReturn => {
   const roleAssignments = unpackMaybes(userAuthInfo?.roleAssignments);
   const communityRoles = useMemo(() => {
     const array = ["community_recruiter", "community_manager"];
-    if (checkRole([ROLE_NAME.PlatformAdmin], roleAssignments)) {
+    if (hasRole([ROLE_NAME.PlatformAdmin], roleAssignments)) {
       array.push("community_admin");
     }
```

Comment:
Should this be replaced with a permission constant?

Reply:
Not here I think, this really isn't dealing in permissions stuff at all

Reply:
Okay, that makes sense to me :+1: